scripts/brc4/brc4_species.pl

```diff
@@ -78,13 +80,25 @@
     $a->{'BRC4.component'} cmp $b->{'BRC4.component'}
       or $a->{'species.scientific_name'} cmp $b->{'species.scientific_name'}
       or $a->{'BRC4.organism_abbrev'} cmp $b->{'BRC4.organism_abbrev'}
-  } @genomes) {
+  } @genomes){
+  # Check if BRC4.organism_abbrev is unique
+  my $abbrev = $genome->{'BRC4.organism_abbrev'};
+  my $prod_name = $genome->{'species.scientific_name'};
+
+  if (defined $abbrev && !$unique_abbrevs{$abbrev}) {
+      # BRC4.organism_abbrev is unique
+      $unique_abbrevs{$abbrev}=$prod_name;
+  }
+  else{
+    die "Error: Non-unique abbreviation encountered: $abbrev $prod_name\n";
+  }
   say join("\t", map { $genome->{$_} // "" } @fields);
-}
+  }
```

Comment:
```suggestion
}
```
I believe this is the right indent.